src/middlewares/analytics.js

```diff
@@ -12,6 +12,8 @@ export default store => next => action => {
     if (shareAnalytics) {
       start(store)
     } else {
+      // TODO add user uuid
+      track('Stopping Analytics')
```

Comment:
I think remove this hack. it does not work because we stop just after. we'll need to figure out a real solution

Reply:
It fires, i can see it in Segment right before it kills the process